### PR TITLE
adds eleven22

### PR DESCRIPTION
**Pull request by redreceipt**

### CODE REVIEW

File: dags/eleven22-rock-content-item-dag.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+from rock.rock_cover_image import fetch_and_save_cover_image
+from rock.rock_content_item_categories import (
+    fetch_and_save_content_item_categories,
+    attach_content_item_categories,
+)
+from rock.rock_features import fetch_and_save_features
+from rock.rock_deleted_content_items_dag import remove_deleted_content_items
+
+
+start_date = datetime(2021, 11, 2)
```
Comment (vinnyjth) [inline dags/eleven22-rock-content-item-dag.py:19]:
```suggestion
start_date = datetime(2021, 11, 15)
```

**Comment by vinnyjth** [comment]
Why are we copying over the entire `create_content_item` function? Is there a difference between the default function and what eleven22 needs?

**Comment by redreceipt** [comment]
no reason. this was just me being lazy and not realizing there was a better way.

**Comment by vinnyjth** [comment]
@redreceipt I'll fix it for yah

**Comment by vinnyjth** [comment]
@redreceipt Where did you put the secrets for Eleven22? I would highly recommend adding them to the `.env` file if you haven't already added them to the production shovel config section.

**Comment by redreceipt** [comment]
nice! I thought i added them to the airflow DB but I forgot about using the env file

**Comment by vinnyjth** [comment]
@redreceipt I'll go ahead and move them to the airflow env file.

**Comment by vinnyjth** [comment]
@redreceipt actually I don't see them in the airflow db :) send them to me and I'll update this pr.

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**